typescript-test-samples/apigw-lambda-sqs-sns-datadog/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "apigw-lambda-sqs-sns-datadog",
+  "version": "0.1.0",
+  "author": "AJ Stuyvenberg",
+  "license": "MIT",
+  "bin": {
+    "apigw-lambda-sqs-sns-datadog": "bin/apigw-lambda-sqs-sns-datadog.js"
+  },
+  "scripts": {
+    "build": "tsc",
+    "watch": "tsc -w",
+    "test": "jest",
+    "cdk": "cdk"
+  },
+  "devDependencies": {
+    "@aws-sdk/client-sns": "^3.321.1",
+    "@types/jest": "^29.4.0",
+    "@types/node": "18.14.6",
+    "aws-cdk": "2.76.0",
+    "jest": "^29.5.0",
+    "ts-jest": "^29.0.5",
+    "ts-node": "^10.9.1",
+    "typescript": "~4.9.5"
```

Comment:
Add `esbuild` as `devDependency` to avoid Docker requirement.

https://docs.aws.amazon.com/cdk/api/v1/docs/aws-lambda-nodejs-readme.html#local-bundling